app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/singleitemcheckout/SingleItemCheckoutDestination.kt

```diff
@@ -204,9 +341,81 @@ private fun ColumnScope.CheckoutMethods(
   }
 }
 
+@Composable
+internal fun SingleItemCheckoutInfoBottomSheet(
+  onDismiss: () -> Unit,
+  sheetState: SheetState,
+  explanationTitleAndText: Pair<String, String>,
+) {
+  ModalBottomSheet(
```

Comment:
Can you check if you could use HedvigBottomSheet here?
It contains some important changes around the dismiss click and letting the sheet slide away before it actually disappears here https://github.com/HedvigInsurance/android/blob/73696b860d2a86faee18d4b89f56f9877714d182/app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/bottomsheet/HedvigBottomSheet.kt#L77-L83
Otherwise if we call dismiss directly it just disappears instantly without animating away. Not sure if it is applicable here but worth a look I think.

Reply:
It also adds the dismiss button in the right spot automatically I think, as a bonus :D